extensions/vertx-web/runtime/pom.xml

```diff
@@ -23,14 +23,8 @@
             <artifactId>quarkus-vertx</artifactId>
         </dependency>
         <dependency>
-            <groupId>io.vertx</groupId>
-            <artifactId>vertx-web</artifactId>
-            <exclusions>
-                <exclusion>
-                    <groupId>com.fasterxml.jackson.core</groupId>
-                    <artifactId>jackson-databind</artifactId>
-                </exclusion>
-            </exclusions>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx-web-common</artifactId>
```

Comment:
Looking at the content, it should probably be named `vertx-http` as it provides the integration with HTTP.

Reply:
(so it's not only web)

Reply:
Done.